l10n_eu_product_adr/security/ir.model.access.csv

```diff
@@ -0,0 +1,9 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_adr_goods_user,access_adr_goods,model_adr_goods,stock.group_stock_user,1,0,0,0
+access_adr_goods_manager,access_adr_goods,model_adr_goods,base.group_erp_manager,1,1,1,1
+access_adr_class_user,access_adr_class,model_adr_class,stock.group_stock_user,1,0,0,0
+access_adr_class_manager,access_adr_class,model_adr_class,base.group_erp_manager,1,1,1,1
+access_adr_label_user,access_adr_label,model_adr_label,stock.group_stock_user,1,0,0,0
+access_adr_label_manager,access_adr_label,model_adr_label,base.group_erp_manager,1,1,1,1
+access_adr_packing_instruction_user,access_adr_packing_instruction,model_adr_packing_instruction,stock.group_stock_user,1,0,0,0
```

Comment:
Could the read access be set on the `base.group_user` instead ?
I see access error for non inventory user opening product pages.
Same question for all read access in this file.

Reply:
Thanks for your comment, I'll check how to prevent the access error.

Reply:
Indeed, we face some access errors when encoding an SO because of this new data model `adr.label`. @StefanRijnhart did you check it or you prefer we open a PR on your branch?

Reply:
@sebalix I have a fix in mind for the product form, but if you run into problems creating an SO then it would be interesting to have a traceback.

Reply:
Fix pushed. Rather than extending the access rights, I've added groups on the product views to prevent the accdess error. I also apply `compute_sudo` to the computation of stock move's adr points, I hope this helps when validating a sales order.